inference-engine/src/vpu/common/src/ngraph/transformations/merge_gather_gather_elements.cpp

```diff
@@ -0,0 +1,145 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "vpu/ngraph/transformations/merge_gather_gather_elements.hpp"
+
+#include <vpu/ngraph/operations/exp_gather_elements.hpp>
+#include <vpu/ngraph/utilities.hpp>
+
+#include <ngraph/pattern/op/wrap_type.hpp>
+#include <ngraph/opsets/opset6.hpp>
+#include <ngraph/pattern/matcher.hpp>
+#include <ngraph/pattern/op/label.hpp>
+#include <ngraph/validation_util.hpp>
+#include <ngraph/log.hpp>
+
+#include <numeric>
+
+NGRAPH_RTTI_DEFINITION(vpu::MergeGatherGatherElements, "MergeGatherGatherElements", 0);
+
+namespace vpu {
+
+bool MergeGatherGatherElements::run_on_function(std::shared_ptr<ngraph::Function> f) {
```

Comment:
@GlebKazantaev Hi, could you please help and take a quick look at the transformation?